docs/src/site/twiki/EntitySLAAlerting.twiki

```diff
@@ -0,0 +1,57 @@
+---++Entity SLA Alerting
+
+Falcon supports SLA in feed and process.
+
+Types of SLA supported for feed:
+
+   1.slaLow
+   1.slaHigh
+
+To know more about feedSla look at [[EntitySpecification][Feed Specification]]
+
+Types of SLA supported for process :
+
+   1.shouldStartIn
+   1.shouldEndIn
+
+To know more about processSla look at [[EntitySpecification][Process Specification]]
+
+Falcon Entity Alerting service do the following things :
+
+   1.Monitor instances of feed and process and send notifications to all the listeners attached to it.
+   1.In case of feed it notifies when an *slaHigh* miss happens. slaLow is not supported.
```

Comment:
Here and in next line, why are you bothering to mention slaLow and shouldStartIn (and above) if they aren't supported?